.github/dependabot.yml

```diff
@@ -0,0 +1,54 @@
+version: 2
+updates:
+- package-ecosystem: bundler
+  directory: "/"
+  schedule:
+    interval: daily
+    time: "03:00"
+    timezone: Europe/London
+  open-pull-requests-limit: 10
```

Comment:
@stephenrichards if dependabot defaults to 10 for this. Does it still need to be specified in the YAML?

Reply:
This is what was generated automatically when I said convert to the new system.  So let's try this, and then we can start tweaking.